openqa-label-known-issues-and-investigate-hook

```diff
@@ -10,4 +10,14 @@ id="${1:?"Need 'job_id'"}"
 host="${host:-"openqa.opensuse.org"}"
 scheme="${scheme:-"https"}"
 host_url="$scheme://$host"
-echo "$host_url/tests/$id" | "$(dirname "$0")"/openqa-label-known-issues 3>&1 1>/dev/null 2>&3- | sed -n 's/\(\S*\) : Unknown issue, to be reviewed.*$/\1/p' | "$(dirname "$0")"/openqa-investigate
+dirname=$(dirname "$0")
+script_dir=${script_dir:-$dirname}
+
+investigate-and-bisect() {
+    read -r test
+    echo "$test" | "$script_dir/openqa-investigate"
+    "$script_dir/openqa-trigger-bisect-jobs" --url "$test"
```

Comment:
Looks like `openqa-trigger-bisect-jobs` will break jobs with directly chained dependencies as it passes `--skip-chained-deps` as `openqa-clone-job` argument. I suppose  `openqa-trigger-bisect-jobs` needed to be adjusted similar to `openqa-investigate` until we can use it in production for jobs with directly chained and parallel dependencies (see https://progress.opensuse.org/issues/95783#note-39). So it would make sense to add an early-abort in `openqa-trigger-bisect-jobs` (or before we call it) for such jobs (similar to `openqa-trigger-bisect-jobs`).

Reply:
> So it would make sense to add an early-abort in openqa-trigger-bisect-jobs (or before we call it) for such jobs

How can I find out if it is "such a job"?

Reply:
You mean this check?
```
'(.job.children["Parallel"] | length) + (.job.parents["Parallel"] | length) + (.job.children["Directly chained"] | length) + (.job.parents["Directly chained"] | length)'
```

Reply:
Yes, this check.

Reply:
#142

Reply:
#142 merged